DirectProgramming/DPC++FPGA/Tutorials/Features/scheduler_target_fmax/src/scheduler_target_fmax.cpp

```diff
@@ -0,0 +1,164 @@
+//==============================================================
+// Copyright Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+#include <CL/sycl.hpp>
+#include <sycl/ext/intel/fpga_extensions.hpp>
+
+// dpc_common.hpp can be found in the dev-utilities include folder.
+// e.g., $ONEAPI_ROOT/dev-utilities/include/dpc_common.hpp
+#include "dpc_common.hpp"
+
+using namespace sycl;
+
+constexpr unsigned seed = 1313;
```

Comment:
Might be better to  use kSeed here since we are "naming" a magic constant value.

Reply:
Addressed.